### ISSUE TITLE
Steam Controller/On Screen Keyboard cannot type capital letters

### ISSUE THREAD

**Issue by jordanhalase** [bevyengine/bevy#24219]
## Bevy version and features

- 0.19-dev 7dda2bc7fcd29e801752346c811484515226a2a9
- Happens in feathers_gallery and ime_support

## \[Optional\] Relevant system information

- Rust 1.95 stable
- Windows 11

## What you did

Select a text input, then press B to open the on-screen keyboard.

## What went wrong

Typing capital letters or shifted letters would not be inputted.

## Additional information

The examples are running as normal Windows apps, **not** through Steam, but Steam is open in the background for on-screen keyboard support.

https://github.com/user-attachments/assets/7a6bede1-a467-4257-97e1-06cb1e885093

**Comment by jordanhalase** [comment]
Note: The keyboard_input_events example logs attempts to type "J" and "#" as

```
2026-05-09T20:38:23.513841Z  INFO keyboard_input_events: KeyboardInput { key_code: Unidentified(Windows(0)), logical_key: Unidentified(Windows(231)), state: Pressed, text: Some("J"), repeat: false, window: 65v0 }
2026-05-09T20:38:40.238963Z  INFO keyboard_input_events: KeyboardInput { key_code: Unidentified(Windows(0)), logical_key: Unidentified(Windows(231)), state: Pressed, text: Some("#"), repeat: true, window: 65v0 }
```

Lowercase "a" as

`2026-05-09T20:41:14.560090Z  INFO keyboard_input_events: KeyboardInput { key_code: KeyA, logical_key: Character("a"), state: Released, text: None, repeat: false, window: 65v0 }`
